src/constants.ts

```diff
@@ -1,3 +1,92 @@
 // The DNS for cdn.jsdelivr.net and fastly.jsdelivr.net have been polluted by the GFW. 
 // For now, gcore.jsdelivr.net is still accessible, but its future availability is uncertain. 
 export const IMG_CDN_URL_PREFIX = 'https://gcore.jsdelivr.net/gh/apache/incubator-kvrocks-website@main/static/img/';
+
+export const userLogoFirstLine = [
```

Comment:
Could the first/second splitting be computed automatically instead of constant definition?
IMHO it is ugly.

Reply:
You are right! Here are some issues. The initial consideration was that we wanted to put some well-known companies in the first row. In fact, obtaining the segmentation according to the order of users can also achieve this effect (provided that the maintainer arranges the users in advance according to some rules).

Reply:
solved.